src/Components/TabsItem/TabsItem.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import "./TabsItem.scss";
+import Icon from "../Icon/Icon";
+
+function TabsItem({ title, content, icon }) {
+  const [isSelected, setIsSelected] = useState(false);
+
+  const handleContentVisibility = () => {
+    setIsSelected(!isSelected);
+  };
+
+  return (
+    <>
+      <div onClick={handleContentVisibility} className="tab-title">
+        <i className="tab-icon">
+          <Icon icon={icon} />
+        </i>
+        <h1 className="tab-heading">{title}</h1>
```

Comment:
i think this is inside a loop, am i right? if yes so it means you have multiple h1 which is wrong. h1 should be just one time in a page.

Reply:
Done.